zebra-state/src/service/finalized_state.rs

```diff
@@ -41,8 +41,15 @@ impl FinalizedState {
             rocksdb::ColumnFamilyDescriptor::new("sprout_nullifiers", db_options.clone()),
             rocksdb::ColumnFamilyDescriptor::new("sapling_nullifiers", db_options.clone()),
         ];
-        let db = rocksdb::DB::open_cf_descriptors(&db_options, path, column_families)
-            .expect("database path and options are valid");
+        let db_check = rocksdb::DB::open_cf_descriptors(&db_options, path, column_families);
+
+        let db = match db_check {
+            Ok(d) => d,
+            Err(_) => panic!(
+                "{} {}",
+                "LOCK file already in use.", "Check if there is antoher zebrad process running."
```

Comment:
We can't assume that the error is "LOCK file already in use":
```suggestion
            Err(e) => panic!(
                "Opening database failed: {:?}. \
                 Hint: Check if there is another zebrad process running.",
                 e,
```